fragments/infra-boot.sh

```diff
@@ -198,6 +198,11 @@ function clone_openshift_ansible() {
 # =============================================================================
 # MAIN
 # =============================================================================
+function sudo_enable_from_ssh() {
+    sed -i "/requiretty/s/^/#/" /etc/sudoers
+}
+
+sudo_enable_from_ssh
```

Comment:
Can this be in cloud-init for the infra-server? (or all?) It takes no parameters and it's universal now.

I understand cloud-init is very full for infra.